task/tkn-bundle/0.1/tkn-bundle.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "image-build, appstudio, hacbs"
+  name: tkn-bundle
+spec:
+  description: |-
+    Creates and pushes a Tekton bundle containing the specified Tekton YAML files.
+  params:
+  - description: Reference of the image task will produce.
+    name: IMAGE
+    type: string
+  - default: .
+    description: Path to the directory to use as context.
+    name: CONTEXT
+    type: string
+  results:
+  - description: Digest of the image just built
+    name: IMAGE_DIGEST
+  - description: Image repository where the built image was pushed
+    name: IMAGE_URL
+  steps:
+  - image: quay.io/redhat-appstudio/appstudio-utils:4580b3ba3012095ff3981e50b6bbf753d4afd4c3
+    name: build
+    script: |
+      #!/bin/env bash
+
+      set -o errexit
+      set -o pipefail
+      set -o nounset
+
+      readarray FILES < <(find $(params.CONTEXT) \( -name '*.yaml' -o -name '*.yml' \))
```

Comment:
Won't this pickup unwanted files? For instance, if executed on the path `task/tkn-bundle/0.1`, it will pick up the test yaml files. Isn't that a problem?

Reply:
Could be addressed, let me see

Reply:
Introduced path negation in `CONTEXT`, see README.md for details